_includes/pbc/all/install-features/202410.0/install-the-category-management-feature.md

```diff
@@ -714,14 +714,14 @@ Make sure that *category-node* and *category-tree* synchronization plugins works
 3. Run the `console sync:data -r category_tree` command.
 4. Check that, in your system, the storage entries are displayed with the `kv:category_node:{store}:{locale}:{id}` and `kv:category_tree:{store}:{locale}:{id}` masks.
 
-Make sure that, when a category is created or edited through ORM, it's exported to Redis and Elasticsearch accordingly.
+Make sure that, when a category is created or edited through ORM, it's exported to key-value storage (Redis or Valkey) and Elasticsearch accordingly.
 
 | STORAGE TYPE  | TARGET ENTITY | EXAMPLE EXPECTED DATA IDENTIFIER |
 |---------------|---------------|----------------------------------|
 | Elasticsearch | CategoryNode  | category_node:at:en_us:2         |
-| Redis         | CategoryNode  | category_node:de:de_de:5         |
-| Redis         | CategoryTree  | category_tree:de:en_us           |
-| Redis         | CategoryImage | category_image:de_de:15          |
+| Redis or Valkey         | CategoryNode  | category_node:de:de_de:5         |
```

Comment:
a note. Mb we can use smth generic like "Key-value storage" everywhere instead of "Redis or Valkey"?